docs/pages/repo/docs/core-concepts/monorepos/ignoring.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Ignoring tasks
+description: Never do the same work twice.
+---
+
+# Ignoring Tasks
+
+[Build caching](/repo/docs/core-concepts/caching) can dramatically speed up your build tasks - but you can do even better by using `npx turbo-ignore`. If your workspace is unaffected by your code changes, you can completely ignore executing a task altogether.
+
+## Using `turbo-ignore`
+
+To skip unaffected work with `npx turbo-ignore`, all you'll need is to make sure your git history is available.
+
+By default, `turbo-ignore` will run the build task in your current working directory. Under the hood, `turbo-ignore` compares the output of `--dry=json` from your previous commit to your current commit.
+
+Here's an example of the command that it will build:
+
+```bash
+npx turbo run build --filter=@example/web...[3c8387ffd98b751305fe3f0284befdd00cbd4610] --dry=json
+```
+
+Note that this command does not _execute_ the build task. Instead, it checks if a **dry run** of a build is different from the **dry run** of the task from the previous commit.
+
+## Customizing behavior
+
+If you'd like to change the default behavior, there are a few arguments available.
+
+- `--workspace`: Runs `turbo-ignore` in the workspace with the name provided
```

Comment:
I would change this slightly. This is altering the name of the workspace used in the first clause of the filter command - so it's not changing _where_ it's running. Maybe something like:

```suggestion
- `--workspace`: Use the provided workspace for the filter command used by `turbo-ignore`. This defaults to the `name` field in the `package.json` in the directory where `turbo-ignore` is invoked.
```